devops/helm/crunchy-postgres/templates/PGUpgrade.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.pgUpgrade.enabled }}
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PGUpgrade
+metadata:
+  name: {{ .Values.pgUpgrade.name | default (printf "%s-pgupgrade-%v-to-%v" .Values.clusterName .Values.pgUpgrade.fromPostgresVersion .Values.pgUpgrade.toPostgresVersion) }}
+spec:
+  postgresClusterName: {{ .Values.clusterName }}
+  fromPostgresVersion: {{ .Values.pgUpgrade.fromPostgresVersion }}
+  toPostgresVersion: {{ .Values.pgUpgrade.toPostgresVersion }}
+  resources:
+    limits:
+      cpu: {{ .Values.pgUpgrade.resources.limits.cpu | default "1" }}
+      memory: {{ .Values.pgUpgrade.resources.limits.memory | default "1Gi" }}
+    requests:
+      cpu: {{ .Values.pgUpgrade.resources.requests.cpu | default "500m" }}
+      memory: {{ .Values.pgUpgrade.resources.requests.memory | default "512Mi" }}
```

Comment:
Unless we find that there is a need to have resources mentioned explicitly, we can remove them for now.